pkg/Microsoft.NETCore.Platforms/runtime.json

```diff
@@ -195,21 +202,35 @@
             "#import": [ "rhel.7" ]
         },
         "rhel.7.0-x64": {
-            "#import": [ "rhel.7", "rhel.7-x64" ]
+            "#import": [ "rhel.7.0", "rhel.7-x64" ]
```

Comment:
Are these changes consistent with what we're auto-generating in 2.x?  Did you consider back-porting the generator code?

Reply:
They are consistent, yes. I didn't want to do a full copy of 2.0/master because i didn't want to worry about things like Android, alpine, etc. In theory those should be fine, but I opted for the minimal change rather than add to much since it's servicing.